src/lib.rs

```diff
@@ -124,6 +124,9 @@ use std::boxed::Box;
 #[cfg(not(feature = "std"))]
 use alloc::boxed::Box;
 
+/// Error type that allows custom implementations of the Serialize trait.
+pub use ser::Error;
```

Comment:
I'm not sure about this one. Serialization is a separate part, and if ser::Error` was to be in root namespace then I guess everything from `ser::*` should.